diff --git a/../rddmanager_2_perm/node_modules/w-web-perm/src/components/LayoutContentGrups.vue "b/./w-web-perm\351\232\250\351\231\204\350\263\207\346\226\231/src/components/LayoutContentGrups.vue"
index f6b9ad4..e1f5076 100644
--- a/../rddmanager_2_perm/node_modules/w-web-perm/src/components/LayoutContentGrups.vue
+++ "b/./w-web-perm\351\232\250\351\231\204\350\263\207\346\226\231/src/components/LayoutContentGrups.vue"
@@ -306,13 +306,13 @@ export default {
             isEditable: false,
             isModified: false,
 
-            tabKeys: [
+            tabKeys: [ //關聯欄之欄序: 本項自己管控的(管控使用權限)在左, 由上層帶過來的(管控所屬使用者)在右
                 'id',
                 'name',
                 'description',
                 'from',
-                'belongUsers',
                 'cpemis',
+                'belongUsers',
                 'userId',
                 'timeCreate',
                 'userIdUpdate',
@@ -322,8 +322,8 @@ export default {
                 'name',
                 'description',
                 'from',
-                'belongUsers',
                 'cpemis',
+                'belongUsers',
                 'userId',
                 'timeCreate',
                 'userIdUpdate',
@@ -333,8 +333,8 @@ export default {
                 'name',
                 'description',
                 // 'from',
-                'belongUsers',
                 'cpemis',
+                'belongUsers',
                 // 'userId',
                 // 'timeCreate',
                 // 'userIdUpdate',
