diff --git a/../rddmanager_2_perm/node_modules/w-web-perm/src/components/LayoutContentPemis.vue "b/./w-web-perm\351\232\250\351\231\204\350\263\207\346\226\231/src/components/LayoutContentPemis.vue"
index 524b4bd..77f32c9 100644
--- a/../rddmanager_2_perm/node_modules/w-web-perm/src/components/LayoutContentPemis.vue
+++ "b/./w-web-perm\351\232\250\351\231\204\350\263\207\346\226\231/src/components/LayoutContentPemis.vue"
@@ -306,13 +306,13 @@ export default {
             isEditable: false,
             isModified: false,
 
-            tabKeys: [
+            tabKeys: [ //關聯欄之欄序: 本項自己管控的(管控對象)在左, 由上層帶過來的(管控所屬權限群組)在右
                 'id',
                 'name',
                 'description',
                 'from',
-                'belongGrups',
                 'crules',
+                'belongGrups',
                 'userId',
                 'timeCreate',
                 'userIdUpdate',
@@ -322,8 +322,8 @@ export default {
                 'name',
                 'description',
                 'from',
-                'belongGrups',
                 'crules',
+                'belongGrups',
                 'userId',
                 'timeCreate',
                 'userIdUpdate',
@@ -333,8 +333,8 @@ export default {
                 'name',
                 'description',
                 // 'from',
-                'belongGrups',
                 'crules',
+                'belongGrups',
                 // 'userId',
                 // 'timeCreate',
                 // 'userIdUpdate',
